wpeverest/everest-forms
# Enhancement - Form submission message display location options

### All Submissions:






### Changes proposed in this Pull Request:
This PR introduces enhancement in form submission message display. The user can set where the message is displayed.


### How to test the changes in this Pull Request:

1. Go-to free version of Everest-forms. Create a form and goto the settings of the form.
2. Choose one of location in  "Success message display option" using the dropdown in Setitngs>Confirmation and save the settings.
3. Test where the success message appears by submitting the form. You can submit the form by embedding it in any of the pages.

### Types of changes:




### Other information:




### Changelog entry

Enhancement - Form submission message display options.
